npm/private/npm_package_store.bzl

```diff
@@ -191,13 +191,12 @@ def _npm_package_store_impl(ctx):
             virtual_store_directory = src_directory
         else:
             virtual_store_directory = ctx.actions.declare_directory(virtual_store_directory_path)
-            hardlink = ctx.attr.src[NpmPackageInfo].hardlink if hasattr(ctx.attr.src[NpmPackageInfo], "hardlink") else False
             copy_directory_bin_action(
                 ctx,
                 src = src_directory,
                 dst = virtual_store_directory,
                 copy_directory_bin = ctx.toolchains["@aspect_bazel_lib//lib:copy_directory_toolchain_type"].copy_directory_info.bin,
-                hardlink = "on" if hardlink else ctx.attr.hardlink,
+                hardlink = ctx.attr.hardlink,
```

Comment:
Is it worth a comment here referencing the bug being fixed and noting that `hardlink` can have potential issues? Or a comment in the docs for `hardlink`?

Partially just so we have something in rules_js referencing the issue, if not a test then a comment in the effected code? 🤷

Reply:
Yes. I'll add some comments here and to the hardlinks docstring.